flutter-studio/testSrc/io/flutter/tests/gui/NewProjectTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2017 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.tests.gui;
+
+import com.android.tools.idea.tests.gui.framework.FlutterGuiTestRule;
+import com.android.tools.idea.tests.gui.framework.GuiTestRunner;
+import com.android.tools.idea.tests.gui.framework.fixture.EditorFixture;
+import com.android.tools.idea.tests.gui.framework.fixture.newProjectWizard.FlutterProjectStepFixture;
+import com.android.tools.idea.tests.gui.framework.fixture.newProjectWizard.FlutterSettingsStepFixture;
+import com.android.tools.idea.tests.gui.framework.fixture.newProjectWizard.NewFlutterProjectWizardFixture;
+import io.flutter.module.FlutterProjectType;
+import io.flutter.tests.util.WizardUtils;
+import org.fest.swing.exception.WaitTimedOutError;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * If flakey tests are found try adjusting these settings:
+ * Settings festSettings = myGuiTest.robot().settings();
+ * festSettings.delayBetweenEvents(50); // 30
```

Comment:
Is this comment still relevant, and, should we just make these delays the default?